website/docs/r/is_virtual_endpoint_gateway.html.markdown

```diff
@@ -88,6 +88,7 @@ Review the argument references that you can specify for your resource.
   **&#x2022;** For more information, about creating access tags, see [working with tags](https://cloud.ibm.com/docs/account?topic=account-tag&interface=ui#create-access-console).</br>
   **&#x2022;** You must have the access listed in the [Granting users access to tag resources](https://cloud.ibm.com/docs/account?topic=account-access) for `access_tags`</br>
   **&#x2022;** `access_tags` must be in the format `key:value`.
+- `allow_dns_resolution_binding` - (Optional, bool) Indicates whether to allow this endpoint gateway to participate in DNS resolution bindings with a VPC that has dns.enable_hub set to true.
```

Comment:
Note - This feature is a Select Location availability invitation only feature. If used in other regions may lead to inconsistencies in state management.

Reply:
<img width="884" alt="image" src="https://github.com/ibm-vpc/terraform-provider-ibm/assets/78945437/8f1837af-c15b-47a7-abb7-c703dc3725cb">